server/routes/appointments/index.js

```diff
@@ -1,6 +1,9 @@
 const router = require('express').Router();
+const { appointmentsSearch } = require('../../controllers');
 const { getAppointmentsByDate } = require('../../controllers');
 
+router.get('/search', appointmentsSearch);
+router.get('/:date');
 router.get('/search');
```

Comment:
Don't repeat routes and route `/:date` is used for what?

Reply:
It's from file structure !!!